modules/calcite/src/main/java/org/apache/ignite/internal/processors/query/calcite/schema/IgniteTableImpl.java

```diff
@@ -48,13 +48,16 @@
 import org.apache.ignite.internal.processors.query.calcite.trait.RewindabilityTrait;
 import org.apache.ignite.internal.processors.query.calcite.type.IgniteTypeFactory;
 import org.apache.ignite.internal.table.TableImpl;
+import org.apache.ignite.lang.IgniteUuid;
 import org.apache.ignite.table.Tuple;
 import org.jetbrains.annotations.Nullable;
 
 /**
  * Ignite table implementation.
  */
 public class IgniteTableImpl extends AbstractTable implements InternalIgniteTable {
+    private final IgniteUuid id;
```

Comment:
we already have _id_ here : TableImpl table;

Reply:
fixed